web/src/api/auth/delete-account.ts

```diff
@@ -0,0 +1,49 @@
+import { Session, getSession, withApiAuthRequired } from "@auth0/nextjs-auth0";
```

Comment:
we will need to combine this with a request to delete your entire user account for when users request user account deletion

Reply:
What are these cases? Should we do this in this PR?

Reply:
later PR